app/controllers/shelter_reviews_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ShelterReviewsController < ApplicationController
+  def new
+    @shelter = Shelter.find(params[:id])
+  end
+
+  def create
+    @shelter = Shelter.find(params[:id])
```

Comment:
We should move this to inside the sad_path line21 so it doesn't make unnecessary db calls.

Reply:
Good call. I went ahead and made that change. The tests are all still passing so the sad path should still be functional in this configuration.